packages/list/index.js

```diff
@@ -50,7 +50,7 @@ export default class List extends Component {
     this.foundation_.setSingleSelection(singleSelection);
     this.foundation_.setWrapFocus(wrapFocus);
     this.foundation_.setVerticalOrientation(this.props[ARIA_ORIENTATION] === VERTICAL);
-    if (selectedIndex) {
+    if (Number.isInteger(selectedIndex)) {
```

Comment:
`isInteger` is not supported by IE11, which is something we need to support. Try using the `typeof selectedNumber === 'number' && isNaN(selectedNumber)` check. I believe `typeof NaN` returns 'number', which is why we need the last part of that statement. 

Also please add a few unit tests to cover the case you mentioned in your first comment of this PR. 

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/isInteger

Reply:
Oh I mistook it for typescript.
I will change it and write some test code!